.colab/00_api_basics.ipynb

```diff
@@ -49,7 +49,7 @@
             "id": "12c5277f",
             "metadata": {},
             "source": [
-                "![LineaPy_artifact](https://github.com/LineaLabs/lineapy/blob/main/examples/tutorials/lineapy_artifact.png?raw=1)"
```

Comment:
This looks very flaky, lets discuss how we can store images centrally so that they are not affected by such moves.

Reply:
for now, dont change this url. looks like its the same in main?

Reply:
Shardul and I synced and decided to move common docs image assets into S3.